Error: incompatible pointer type

Hi,

I have encountered the following error when I tried to compile this project using the option `--with-nfqueue` on OpenWRT env:
```sh
dhcp.c: In function 'nfqueue_cb_in':
dhcp.c:886:3: error: passing argument 2 of 'nfq_get_payload' from incompatible pointer type [-Werror]
   ret = nfq_get_payload(nfa, &data);
   ^
In file included from net.h:37:0,
                 from tun.h:26,
                 from chilli.h:27,
                 from dhcp.c:22:
/home/openwrt/15.05/staging_dir/target-mips_34kc_uClibc-0.9.33.2/usr/include/libnetfilter_queue/libnetfilter_queue.h:119:12: note: expected 'unsigned char **' but argument is of type 'char **'
 extern int nfq_get_payload(struct nfq_data *nfad, unsigned char **data);
            ^
dhcp.c: In function 'nfqueue_cb_out':
dhcp.c:942:3: error: passing argument 2 of 'nfq_get_payload' from incompatible pointer type [-Werror]
   ret = nfq_get_payload(nfa, &data);
   ^
In file included from net.h:37:0,
                 from tun.h:26,
                 from chilli.h:27,
                 from dhcp.c:22:
/home/openwrt/15.05/staging_dir/target-mips_34kc_uClibc-0.9.33.2/usr/include/libnetfilter_queue/libnetfilter_queue.h:119:12: note: expected 'unsigned char **' but argument is of type 'char **'
 extern int nfq_get_payload(struct nfq_data *nfad, unsigned char **data);
            ^
```

I can fix these errors using these patches:
```patch
--- a/src/dhcp.c
+++ b/src/dhcp.c
@@ -865,7 +865,7 @@
   struct nfqnl_msg_packet_hdr *ph;
   struct nfqnl_msg_packet_hw *hw;
   u_int32_t id = 0;
-  char *data;
+  unsigned char *data;
   int ret;
 
   int result = NF_DROP;
@@ -928,7 +928,7 @@
 		   struct nfq_data *nfa, void *cbdata) {
   struct nfqnl_msg_packet_hdr *ph;
   u_int32_t id = 0;
-  char *data;
+  unsigned char *data;
   int ret;
 
   int result = NF_DROP;

```

```patch
--- a/src/queue.c
+++ b/src/queue.c
@@ -13,7 +13,7 @@
   struct nfqnl_msg_packet_hdr *ph;
   u_int32_t mark,ifi;
   int id = 0;
-  char *data;
+  unsigned char *data;
   int ret;
 
   ph = nfq_get_msg_packet_hdr(tb);

```

--- a/src/dhcp.c
+++ b/src/dhcp.c
@@ -865,7 +865,7 @@ int nfqueue_cb_in(struct nfq_q_handle *qh, struct nfgenmsg *nfmsg,
   struct nfqnl_msg_packet_hdr *ph;
   struct nfqnl_msg_packet_hw *hw;
   u_int32_t id = 0;
-  char *data;
+  unsigned char *data;
   int ret;
 
   int result = NF_DROP;
@@ -928,7 +928,7 @@ int nfqueue_cb_out(struct nfq_q_handle *qh, struct nfgenmsg *nfmsg,
 		   struct nfq_data *nfa, void *cbdata) {
   struct nfqnl_msg_packet_hdr *ph;
   u_int32_t id = 0;
-  char *data;
+  unsigned char *data;
   int ret;
 
   int result = NF_DROP;

--- a/src/queue.c
+++ b/src/queue.c
@@ -13,7 +13,7 @@ static u_int32_t print_pkt (struct nfq_data *tb) {
   struct nfqnl_msg_packet_hdr *ph;
   u_int32_t mark,ifi;
   int id = 0;
-  char *data;
+  unsigned char *data;
   int ret;
 
   ph = nfq_get_msg_packet_hdr(tb);

